### PR TITLE
Add: User Feedback assets

### PR DESCRIPTION
**Pull request by jmanhart**
Screen Recording 2019-08-29 at 01 21 PM

Adding in the assets for the user feedback empty state.

### CODE REVIEW

File: storybook-static/index.html [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<!doctype html><html><head><meta charset="utf-8"><title>Storybook</title><meta name="viewport" content="width=device-width,initial-scale=1"><style>html, body {
```
Comment (Chrissy) [inline storybook-static/index.html]: Can we figure out where this file came from and if we need it?

File: storybook-static/sb_dll/storybook_ui_dll.LICENCE [hunk 2]
```diff
@@ -0,0 +1,115 @@
+/**!
```
Comment (Chrissy) [inline storybook-static/sb_dll/storybook_ui_dll.LICENCE]: Same here

File: yarn.lock [hunk 3]
```diff
@@ -4557,15 +4550,6 @@ gulp-cli@^2.2.0:
     v8flags "^3.0.1"
     yargs "^7.1.0"
 
-gulp-concat@^2.6.1:
```
Comment (Chrissy) [inline yarn.lock:4560]: this is good—let's keep this.

File: styles/user-feedback.css [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+  #chat-box {
+    animation: 0.5s animate-top-left-in cubic-bezier(0.68, -0.55, 0.265, 1.55);
+    transform-origin: center center;
+  }
+  
+  #chat-avatar {
+    animation: 0.5s animate-bottom-right-in cubic-bezier(0.68, -0.55, 0.265, 1.55);
+    transform-origin: center center;
+  }
+
```
Comment (Chrissy) [inline styles/user-feedback.css]: so much whitespace lol

**Review by Chrissy — CHANGES_REQUESTED** [review]
code looks great! can't wait to see the designs

**Review by Chrissy — APPROVED** [review]
post a screenshot to the description I wanna see it!!!

**Comment by maheskett** [comment]
Hm do you know why the fire emoji colors look muted here compared to figma? Also, @Chrissy do we need to bump the version to have access in sentry?

**Comment by Chrissy** [comment]
I just assumed that was the screenshot tool. @jmanhart can you verify if that is happening in dev?

and yeah, thank you for reminding me. we should bump the version to 2.0.1.

**Comment by jmanhart** [comment]
@Chrissy The flames appear muted in my build as well. I updated the Figma file to have no flames now and it should all be better now.

**Pull request merged**